mtags/src/main/scala-2/scala/meta/internal/pc/HoverProvider.scala

```diff
@@ -175,8 +204,18 @@ class HoverProvider(val compiler: MetalsGlobal, params: OffsetParams) {
       range: Position
   ): Option[Hover] = {
     if (tpe == null || tpe.isErroneous || tpe == NoType) None
-    else if (symbol == null || symbol == NoSymbol || symbol.isErroneous) None
-    else if (tpe.typeSymbol.isAnonymousClass) None
+    else if (symbol == null || symbol == NoSymbol || symbol.isErroneous) {
+      val context = doLocateContext(pos)
```

Comment:
Is this change needed for `type on selection` ? If not I would move it to a separate PR

Reply:
not necessary, removed